.buildkite/scripts/packaging/package.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+source .buildkite/scripts/packaging/package-util.sh
+
+IMG_POSTFIX="-SNAPSHOT"
+VARIANTS=("" "-ubi" "-oss")
+VERSION="$(make get-version)"
+SOURCE_TAG+="${VERSION}${IMG_POSTFIX}"
+TARGET="observability-ci/${BEATS_PROJECT_NAME}"
+
+echo "--- Creating package"
+mage -d "${BEATS_PROJECT_NAME}" package
+
+echo "--- Distribution list"
+dir="${BEATS_PROJECT_NAME}/build/distributions"
+buildkite-agent artifact upload "$dir/*.tar.gz;$dir/*.tar.gz.sha512"
+
+echo "--- Docker image list"
+docker images
+
+define_tags
+
+targetSuffix=""
+if [[ ${HW_TYPE} == "aarch64" || ${HW_TYPE} == "arm64" ]]; then
+  targetSuffix="-arm64"
+fi
+
+for variant in "${VARIANTS[@]}"; do
+  source="beats/${BEATS_PROJECT_NAME}${variant}"
+
+  for tag in "${tags[@]}"; do
+    targetTag=$tag${targetSuffix}
+
+    sourceName="${DOCKER_REGISTRY}/${source}:${SOURCE_TAG}"
+    targetName="${DOCKER_REGISTRY}/${TARGET}:${targetTag}"
+    # Remove following line once beats fully migrated
```

Comment:
can we made this a `# TODO` and additionally say that this suffix is to avoid clashing with the Jenkins produced images, until the migration is over (i.e. what you mentioned [here](https://github.com/elastic/beats/pull/37827/files#r1530439253))